Sources/AutoLayout/Constrain.swift

```diff
@@ -41,6 +40,7 @@ public final class ConstraintGroup {
 public func constrain<A: LayoutItem>(
     _ a: A,
     replacing group: ConstraintGroup = .init(),
+    isActive: Bool = true,
```

Comment:
Maybe call this new parameter `activate`? (or `activated`) 💄 

My reasoning is that `isActive` refers to the group property, not the constrain API (which creates/updates a group)

Reply:
Makes sense, I'll go with one of those